interface/billing/edih_view.php

```diff
@@ -29,276 +29,302 @@
 <html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
 
 <head>
-    <title><?php echo xlt("edi history"); ?></title>
+    <title><?php echo xlt("EDI History"); ?></title>
     <meta http-equiv="content-type" content="text/html;charset=utf-8" />
 
-    <!-- TODO: Address no_bootstrap here !-->
-    <?php Header::setupHeader(['no_main-theme', 'no_bootstrap', 'datetime-picker', 'datatables', 'datatables-jqui', 'datatables-jqui-theme', 'datatables-scroller', 'datatables-scroller-jqui-theme', 'jquery-ui', 'jquery-ui-sunny']); ?>
+    <?php Header::setupHeader(['no_main-theme', 'datetime-picker', 'datatables', 'datatables-bs', 'datatables-scroller']); ?>
```

Comment:
rec dropping the `no_main-theme` and seeing what happens

Reply:
prob also need this token in there `datatables-dt`, which is generally needed whenever using `datatables-bs`

Reply:
Hi @bradymiller sir

I removed the no_main-theme, and it seems like to apply your custom style. e.g. button color. 
Any suggestions or feedback?

![image](https://user-images.githubusercontent.com/22230889/80299858-56f10100-87ca-11ea-900d-206e0b73e452.png)